docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgpartman.md

```diff
@@ -210,10 +210,6 @@ ERROR:  partman is not a valid partitioning type for pg_partman
 
 The pg_partman `create_parent()` function requires an access exclusive lock on the parent table to create new child partitions. Currently, access exclusive locks are not supported in YugabyteDB, and are disabled in this function.
 
-### ADVISORY LOCKS
```

Comment:
What if you enable pg_partman but don't set the advisory lock flags for the cluster

Reply:
Then the advisory locks calls will fail with the message to enable the preview feature.

Reply:
Looks like there is dev work to get pg_partman to use advisory_locks. Will undo this change for now.